bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx117-na-relay.html

```diff
@@ -0,0 +1,84 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+{{ css_bundle('firefox_whatsnew_117_na_relay') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_header %}
+<header class="c-page-header">
+  <div class="mzp-l-content c-page-header-inner">
+    <h2 class="c-page-header-logo-fx">{{ ftl('whatsnew-firefox') }}</h2>
+    {# aside is used to hide notification content from reader mode #}
```

Comment:
The reader view pages needed this repeated header so the page itself would work better in reader view, but I don't think you need to repeat it on this page. You can remove the whole block and it'll get it from the base template.